### PR TITLE
ES-908324 Modify DocIO GitHub and UG for document processing playground

### PR DESCRIPTION
**Pull request by AkashArul26**
Hi All,

Modify docio gitHub examples repository for document processing playground.

Regards,
Akash.

### CODE REVIEW

File: Word-document/Merge-documents-in-new-page/.NET/Merge-documents-in-new-page/Program.cs [hunk 1]
```diff
@@ -10,20 +10,20 @@ class Program
...
         static void Main(string[] args)
         {
-            using (FileStream sourceStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/SourceDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            using (FileStream sourceStreamPath = new FileStream(Path.GetFullPath(@"Data/SourceDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
             {
                 //Opens an source document from file system through constructor of WordDocument class.
                 using (WordDocument sourceDocument = new WordDocument(sourceStreamPath, FormatType.Automatic))
                 {
-                    using (FileStream destinationStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/DestinationDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+                    using (FileStream destinationStreamPath = new FileStream(@"Data/DestinationDocument.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
```
Comment (LokeshBaskar) [inline Word-document/Merge-documents-in-new-page/.NET/Merge-documents-in-new-page/Program.cs:18]: Why the Path.GetFullPath is removed here

File: Word-document/Merge-documents-in-same-page/.NET/Merge-documents-in-same-page/Program.cs [hunk 2]
```diff
@@ -10,12 +10,12 @@ class Program
...
         static void Main(string[] args)
         {
-            using (FileStream sourceStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/SourceDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+            using (FileStream sourceStreamPath = new FileStream(Path.GetFullPath(@"Data/SourceDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
             {
                 //Opens an source document from file system through constructor of WordDocument class.
                 using (WordDocument sourceDocument = new WordDocument(sourceStreamPath, FormatType.Automatic))
                 {
-                    using (FileStream destinationStreamPath = new FileStream(Path.GetFullPath(@"../../../Data/DestinationDocument.docx"), FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+                    using (FileStream destinationStreamPath = new FileStream(@"Data/DestinationDocument.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
```
Comment (LokeshBaskar) [inline Word-document/Merge-documents-in-same-page/.NET/Merge-documents-in-same-page/Program.cs:18]: Why the Path.GetFullPath is removed here. Please do changes properly

**Review by LokeshBaskar — CHANGES_REQUESTED** [review]
Please address the below feedbacks

File: Word-document/Merge-documents-in-new-page/.NET/Merge-documents-in-new-page/Program.cs (hunk 1 above)
Comment (AkashArul26) [reply Word-document/Merge-documents-in-new-page/.NET/Merge-documents-in-new-page/Program.cs]: Addressed the concerns

File: Word-document/Merge-documents-in-same-page/.NET/Merge-documents-in-same-page/Program.cs (hunk 2 above)
Comment (AkashArul26) [reply Word-document/Merge-documents-in-same-page/.NET/Merge-documents-in-same-page/Program.cs]: Addressed the concerns

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**